tchannel/tornado/connection.py

```diff
@@ -121,7 +120,13 @@ def __init__(self, connection, tchannel=None):
         self._loop_running = False
 
         self.tchannel = tchannel
+        self.incoming_requests = {}
+        self.incoming_responses = {}
```

Comment:
It is place to save the "inflight" incoming requests, but considering we may add something for outgoing req/resp in the future, I'd rather name it incoming